packages/video_player_avplay/tizen/src/drm_manager_proxy.cc

```diff
@@ -6,59 +6,103 @@
 
 #include <dlfcn.h>
 
-FuncDMGRSetData DMGRSetData = nullptr;
-FuncDMGRGetData DMGRGetData = nullptr;
-FuncDMGRSetDRMLocalMode DMGRSetDRMLocalMode = nullptr;
-FuncDMGRCreateDRMSession DMGRCreateDRMSession = nullptr;
-FuncDMGRSecurityInitCompleteCB DMGRSecurityInitCompleteCB = nullptr;
-FuncDMGRReleaseDRMSession DMGRReleaseDRMSession = nullptr;
-
-void* OpenDrmManagerProxy() { return dlopen("libdrmmanager.so.0", RTLD_LAZY); }
-
-int InitDrmManagerProxy(void* handle) {
-  if (!handle) {
-    return DM_ERROR_INVALID_PARAM;
+#include "log.h"
+
+DrmManagerProxy::DrmManagerProxy()
+    : drm_manager_handle_(::dlopen("libdrmmanager.so.0", RTLD_LAZY)) {
```

Comment:
:: doesn't seem to be necessary.
```suggestion
    : drm_manager_handle_(dlopen("libdrmmanager.so.0", RTLD_LAZY)) {
```

Reply:
Ok